Packs/Palo_Alto_Networks_WildFire/Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.yml

```diff
@@ -38,6 +39,11 @@ configuration:
   name: suppress_file_type_error
   required: false
   type: 8
+- display: API Key (Deprecated)
+  name: token
```

Comment:
please add `hidden: true`

Reply:
We can't, I tried. 
Validate will fail on:
```
[IN124] - Parameter: "token" can't be hidden. Please remove this field.
```